jdaviz/configs/default/plugins/virtual_observatory/vo_plugin.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <j-tray-plugin
+    description='Download a data product from the Virtual Observatory'
+    :link="'https://www.ivoa.net/astronomers/index.html'"
+    :popout_button="popout_button">
+    <v-form v-model="all_fields_filled">
+
+      <j-plugin-section-header>Source Selection</j-plugin-section-header>
+
+      <plugin-viewer-select
+        :items="viewer_items"
+        :selected.sync="viewer_selected"
+        label="Viewer"
+        :show_if_single_entry="true"
+        hint="Select a viewer to retrieve center coordinates, or Manual for manual coordinate entry."
+      />
+
+      <v-row v-if="viewer_selected !== 'Manual'">
+        <v-switch
+          v-model="coord_follow_viewer_pan"
+          label="Follow Viewer Center"
+          hint="Automatically adjust coordinates as viewer pans and zooms"
+          persistent-hint
+        ></v-switch>
+      </v-row>
+
+      <v-row>
+        <div :style="!(viewer_selected !== 'Manual' && !coord_follow_viewer_pan) ? 'width: 100%' : 'width: calc(100% - 32px)'">
+          <v-text-field
+            v-model="source"
+            label="Source/Coordinates"
+            hint="Enter a source name or coordinates in degrees to center your query on"
+            :disabled="viewer_selected !== 'Manual'"
+            :rules="[() => !!source || 'This field is required']"
+            persistent-hint>
+          </v-text-field>
+        </div>
+        <div v-if="viewer_selected !== 'Manual' && !coord_follow_viewer_pan" style="line-height:64px; width:32px">
+          <v-btn
+            id="autocenterbtn"
+            @click="center_on_data"
+            :disabled="viewer_centered"
+            icon>
+            <v-icon>
+              {{ viewer_centered ? 'mdi-crosshairs-gps' : 'mdi-crosshairs' }}
+            </v-icon>
+          </v-btn>
+        </div>
+      </v-row>
+
+      <v-row>
+        <v-select
+          v-model="coordframe_selected"
+          :menu-props="{ left: true }"
+          attach
+          :items="coordframes"
+          label="Coordinate Frame"
+          hint="Astronomical Coordinate Frame of the provided Coordinates"
+          :disabled="viewer_selected !== 'Manual'"
+          persistent-hint
+        ></v-select>
+      </v-row>
+
+    <j-plugin-section-header>Common Options</j-plugin-section-header>
```

Comment:
is there room in the future for more in this section or will it remain just the search radius?  Maybe the header can either be more specific ("search radius/terms") or the section just removed and search criteria lumped in with source selection?

Reply:
Good point, this was mainly a relic from when the source selection was combined in the "common options." This header may be useful if we expand this functionality to specviz and cubeviz, but that can be decided later

Done in https://github.com/spacetelescope/jdaviz/pull/2872/commits/518d60cf1a8ce035c6ef87ca0620c3f80a6e1b8e